### PR TITLE
(maint) Get rid of Guac-isms and get ready for direct RDP

### PR DESCRIPTION
**Pull request by greglarkin**

### CODE REVIEW

**Comment by greglarkin** [comment]
@binford2k This PR goes along with https://github.com/puppetlabs/vcmanager/pull/72. I removed that awful hack to support booting the Windows classes into separate "virtual classrooms" by modifying their titles in Learndot.

**Comment by greglarkin** [comment]
I will also have to clean up the AWS configuration (VPCs and subnets) as well, but I'll wait until this is merged and in use.

File: cloud/aws/courses/puppetize-virtual-classroom.yaml [hunk 1]
```diff
@@ -10,12 +10,10 @@ instructorwin:
   type: "t2.medium"
   key_name: "training"
   security_group_ids:
-    - "sg-dc5409a6"
-  subnet_id: "subnet-987484fe"
+    - "sg-4d3f2b31"
```
Comment (greglarkin) [inline cloud/aws/courses/puppetize-virtual-classroom.yaml]: @binford2k The new security group allows RDP access from the ThinRDP server only at the moment. If we need to, we can open it up to public direct RDP connections to the Windows instances.

File: cloud/aws/codes.yaml [hunk 2]
```diff
@@ -10,9 +10,5 @@
...
 "Practitioner": "practitioner"
 "Puppet Essentials for Windows": "windows"
-"Virtual Puppetizing Infrastructure 1": "puppetize-virtual-classroom-1"
-"Virtual Puppetizing Infrastructure 2": "puppetize-virtual-classroom-2"
-"Virtual Puppetizing Infrastructure 3": "puppetize-virtual-classroom-3"
-"Puppetizing Infrastructure 1": "puppetize-virtual-classroom-1"
-"Puppetizing Infrastructure 2": "puppetize-virtual-classroom-2"
-"Puppetizing Infrastructure 3": "puppetize-virtual-classroom-3"
+"Virtual Puppetizing Infrastructure": "puppetize-virtual-classroom"
+"Puppetizing Infrastructure": "puppetize-virtual-classroom"
```
Comment (greglarkin) [inline cloud/aws/codes.yaml]: Looked like the PI course can be named differently in certain cases, so this makes sure that both name variants are able to boot the classroom with the correct machines.

File: cloud/aws/courses/puppetize-virtual-classroom.yaml (hunk 1 above)
Comment (binford2k) [reply cloud/aws/courses/puppetize-virtual-classroom.yaml]: This comment is stale now, right?

Comment (greglarkin) [reply cloud/aws/courses/puppetize-virtual-classroom.yaml]: @binford2k Yes, I opened up the rule in that SG so connections to port 3389 are accepted from anywhere.

Comment (binford2k) [reply cloud/aws/courses/puppetize-virtual-classroom.yaml]: hmm. I must be missing something, I think. The default virtual classroom security group allows RDP from anywhere. Is something else needed?

Comment (greglarkin) [reply cloud/aws/courses/puppetize-virtual-classroom.yaml]: I didn't notice that, I guess. I can update the PR to reference sg-f3b33794 (default virtual classroom) and get rid of the other one with that one rule in it.

Comment (greglarkin) [reply cloud/aws/courses/puppetize-virtual-classroom.yaml]: @binford2k Updated the SG with a new commit

**Pull request merged**